crates/voicevox_core_c_api/tests/e2e/testcases/user_dict_manipulate.rs

```diff
@@ -0,0 +1,224 @@
+// ユーザー辞書の操作をテストする。
+
+use assert_cmd::assert::AssertResult;
+use once_cell::sync::Lazy;
+use std::{
+    ffi::{CStr, CString},
+    mem::MaybeUninit,
+};
+use tempfile::NamedTempFile;
+use voicevox_core::result_code::VoicevoxResultCode;
+
+use libloading::Library;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    assert_cdylib::{self, case, Utf8Output},
+    snapshots,
+    symbols::{Symbols, VoicevoxUserDict, VoicevoxUserDictWord, VoicevoxUserDictWordType},
+};
+
+case!(TestCase);
+
+#[derive(Serialize, Deserialize)]
+struct TestCase;
+
+#[typetag::serde(name = "user_dict_manipulate")]
+impl assert_cdylib::TestCase for TestCase {
+    unsafe fn exec(&self, lib: &Library) -> anyhow::Result<()> {
+        let Symbols {
+            voicevox_default_user_dict_word,
+            voicevox_user_dict_new,
+            voicevox_user_dict_add_word,
+            voicevox_user_dict_update_word,
+            voicevox_user_dict_remove_word,
+            voicevox_user_dict_get_json,
+            voicevox_user_dict_import,
+            voicevox_user_dict_load,
+            voicevox_user_dict_save,
+            voicevox_user_dict_delete,
+            voicevox_user_dict_uuid_free,
+            voicevox_json_free,
+            ..
+        } = Symbols::new(lib)?;
+
+        let get_json = |dict: &*mut VoicevoxUserDict| -> String {
```

Comment:
ここ、`&*mut`にしている意図が掴めなかったので、理由が無ければこうした方がいいと思います。

```suggestion
        let get_json = |dict: *const VoicevoxUserDict| -> String {
```

Reply:
シンプルに、dict作成時の型がそうだったからですね。